src/main/resources/application-local.yaml

```diff
@@ -1,3 +1,5 @@
 server.port=8090
 
 geo-data-server.base.url=http://localhost:8087
+spring.servlet.multipart.max-file-size=-1
```

Comment:
Not sure we need to put this in both application-local.yaml AND config/application.yaml.
It should be ok to put it only in one of them